modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -71,5 +71,13 @@
 			"java",
 			"jsp"
 		]
+	},
+	{
+		"classNames": [
+			"CommerceAccountHelper"
+		],
+		"from": "getCurrentCommerceAccount(HttpServletRequest)",
+		"issueKey": "LPS-196638",
+		"to": "getCurrentAccountEntry(0, param#0#)"
```

Comment:
@alberttgomes please rename this commit message, the name of the file is `replacement.json` and please remove the creation of the LPS-189358 patterns.